src/DynamoPackages/Interfaces/IPackageLoader.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using Dynamo.PackageManager;
+
+namespace DynamoPackages.Interfaces
+{
+    /// <summary>
+    /// This interface is for PackageLoader. It can be expanded if needed,
+    /// currently only exposed inViewLoadedParams and referenced in WorkspaceDependencyViewExtension
+    /// </summary>
+    public interface IPackageLoader
+    {
+        /// <summary>
+        /// List of local packages
```

Comment:
to be part of an interface this needs more description, what if someone else is trying to implement this interface?

Reply:
I guess my question would be do we want to continue use the interface this way similar to how we expose PackageManagerClientViewModel. I believe @aparajit-pratap has some concerns. I would like to have an agreement first, I can totally remove the interface and only expose a list of local packages in the LoadedParam but that can be out of the date quickly and obsolete later.

Reply:
yeah, I'm not sure, @scottmitchell and I talked a lot about adding the other interface which is weirder as the idea was it would contain more UI centric functionality like install,remove - this is just a query, so I'm not sure we want to wrap them up together.


as they are now one can be used from modelExtensions as well, if we put them together we no longer have that flexibility.

Reply:
Ended removing the interface, marking as resolved